x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_related_list.tsx

```diff
@@ -34,16 +36,56 @@ interface MatchingEventEntry {
   setQueryParams: () => void;
 }
 
+const StyledRelatedLimitWarning = styled(RelatedEventLimitWarning)`

```

Comment:
I think it makes sense to apply additional styling to a component in some cases. For example, to help the component fit in the place where you're using it, or other context specific things. However, this seems like a lot. Should much of this styling be inherent to `RelatedEventLimitWarning`? If so, perhaps move it into the same module. You could even define `RelatedEventLimitWarning` with the styles baked in for simplicity.

Reply:
I think everything but the margin _could_ be moved into the component, but these are all "small breaks" to default EUI styling to satisfy the design in this case. I think there is something to be gained by keeping the base component as "EUI"-ish as possible and breaking the shared style concerns at the call sites. I'm happy to change it if you still think you want that, though.